fog/recovery_db_iface/src/types.rs

```diff
@@ -248,8 +248,8 @@ pub struct ReportData {
     /// The ingest invocation id that wrote this report.
     pub ingest_invocation_id: Option<IngestInvocationId>,
 
-    /// The Intel Attestation Service report, which include the pubkey
-    pub report: VerificationReport,
+    /// The enclave attestation evidence, which include the pubkey
+    pub attestation_evidence: AttestationEvidence,
```

Comment:
The DB stores bytes, so it doesn't care what this looks like. Not sure if we want to look into renaming the DB table entry from `report` to something else.